free-courses-ko.md

```diff
@@ -86,9 +86,9 @@
 
 ### Deep Learning
 
-* [모두를 위한 딥러닝 시즌 1](https://www.youtube.com/playlist?list=PLlMkM4tgfjnLSOjrEJN31gZATbcj_MpUm)
-* [모두를 위한 딥러닝 시즌 2](https://www.youtube.com/playlist?list=PLQ28Nx3M4Jrguyuwg4xe9d9t2XE639e5C)
-
+* [C++로 배우는 딥러닝](https://www.youtube.com/watch?v=nHt7BHyJGko&list=PLNfg4W25Tapy5hIBmFZgT5coii1HUX6BD) - 홍성모 (youtube)
```

Comment:
We prefer YouTube URLs to be to the `/playlist` endpoint rather than the `/watch` endpoint. Could you update this so users can get a better overview of the entire course?

```suggestion
* [C++로 배우는 딥러닝](https://www.youtube.com/playlist?list=PLNfg4W25Tapy5hIBmFZgT5coii1HUX6BD) - 홍성모 (youtube)
```